### DISCUSSION TITLE
installed without firmware update. Not working properly need help with solution.

### DISCUSSION THREAD

**Discussion by Prophit219** [OnionUI/Onion#538] [Q&A] [answered]
I did the install without checking the firmware on a new 64gb sandisk card formatted to fat32. It installed alright as I got to the last step and power on the device. I get the Miyoo splash screen followed by the Onion OS title then it just goes blank. If I hold the power button it gives me the Onion OS splash screen for shutting down and turns off. 

Can I just use the old SD card and update the firmware and just reinstall this new one or do I have to do the whole installation over again? Would I need to go back to stock firmware if so how do I do that?

**Comment by Sichroteph** [comment]
No need to reinstall everything. Just update the firmware.

**Comment by Prophit219** [comment, marked as answer]
I guess it was my SD card wouldn't take the update either after format so I just used another new one and it worked
